### PR TITLE
PBM-955 Restore of data-at-rest encrypted data

### PR DESCRIPTION
**Pull request by nastena1606**

### CODE REVIEW

File: docs/features/physical.md [hunk 1]
```diff
@@ -37,23 +37,38 @@ During physical backups and restores, ``pbm-agents`` don’t export / import dat
...
 
 To restore the encrypted data from the backup, do the following:
 
-1. Put the encryption key / specify the path to the key on at least one node of every replica set. 
+1. Configure data-at-rest encryption settings on all nodes of your destination cluster or replica set to match the settings of the target cluster where you made the backup
+
+2. Specify the KMIP server / the path to the master encryption key in the configuration file on every node. 
+
```
Comment (sandraromanchenko) [inline docs/features/physical.md:57]: These 2 steps are the same - just leave the first one

File: docs/features/physical.md [hunk 2]
```diff
@@ -37,23 +37,38 @@ During physical backups and restores, ``pbm-agents`` don’t export / import dat
...
+During the restore, Percona Backup for MongoDB restores the data all nodes using the same master key. To meet the security policy requirements in your organization, we recommend to rotate the master encryption keys afterwards. 
+
+To learn more about master key rotation, refer to the following documentation:
+
+* [Master key rotation in HashiCorp Vault server](https://docs.percona.com/percona-server-for-mongodb/6.0/vault.html#key-rotation)
+* [KMIP master key rotation](https://www.mongodb.com/docs/manual/tutorial/rotate-encryption-key/#kmip-master-key-rotation)
+
+### Restore for Percona Server for MongoDB **before** 4.4.19-19, 5.0.15-13, 6.0.5-4 using HashiCorpVault
+
+If you run the Percona Server for MongoDB version **before** 4.4.19-19, 5.0.15-13, 6.0.5-4, and PBM before 2.0.1, and use the Vault server, consider using the following restore scenario:
```
Comment (sandraromanchenko) [inline docs/features/physical.md:67]: It's technically possible to use instruction above - however after restore it won't be possible to rotate key, we should mention that

**Review by sandraromanchenko — CHANGES_REQUESTED** [review]

File: docs/features/physical.md (hunk 2 above)
Comment (nastena1606) [reply docs/features/physical.md]: Updated the doc

File: docs/features/physical.md [hunk 3]
```diff
@@ -37,23 +37,34 @@ During physical backups and restores, ``pbm-agents`` don’t export / import dat
...
+* [Master key rotation in HashiCorp Vault server](https://docs.percona.com/percona-server-for-mongodb/6.0/vault.html#key-rotation)
+* [KMIP master key rotation](https://www.mongodb.com/docs/manual/tutorial/rotate-encryption-key/#kmip-master-key-rotation)
+
+### Restore for Percona Server for MongoDB **before** 4.4.19-19, 5.0.15-13, 6.0.5-4 using HashiCorpVault
+
+In Percona Server for MongoDB version **before** 4.4.19-19, 5.0.15-13, 6.0.5-4 with the Vault server used for data-at-rest encryption, the master key rotation with the same key is not supported. If you run these versions of Percona Server for MongoDB and PBM before 2.1.0, consider using the scenario where PBM restores the data on one node of every replica set. The remaining nodes receive the data during the initial sync.
```
Comment (sandraromanchenko) [inline docs/features/physical.md:63]:
```suggestion
In Percona Server for MongoDB version **before** 4.4.19-19, 5.0.15-13, 6.0.5-4 with the Vault server used for data-at-rest encryption, the master key rotation with the same key for used for 2+ nodes is not supported. If you run these versions of Percona Server for MongoDB and PBM before 2.1.0, consider using the scenario where PBM restores the data on one node of every replica set. The remaining nodes receive the data during the initial sync. 
```

**Review by sandraromanchenko — CHANGES_REQUESTED** [review]
Small changes, the rest is perfect ^_^

File: docs/features/physical.md (hunk 3 above)
Comment (nastena1606) [reply docs/features/physical.md]: Fixed

**Review by sandraromanchenko — APPROVED** [review]

**Pull request merged**